app/assets/stylesheets/retro.scss

```diff
@@ -0,0 +1,68 @@
+.nav {
+  background-color: #696969;
+  height: 60px;
+}
+
+.headerItems {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.header-icon {
+  border-radius: 30px;
+  background-color: white;
+  padding: 4px;
+  margin-right: 15px;
+  margin-top: 6px;
+}
+
+#time {
+  outline: none;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  font-weight: bold;
+  font-size: 17px;
+  color: #02a397;
+  display: none;
+  margin-right: 44%;
+}
+
+.start-button {
+  margin-right: 42%;
+}
+
+.boards {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: row;
+  height: calc(100% - 60px);
+
+  .section {
+    border-width: 1px;
+    width: 200px;
+    min-height: 300px;
+    height: auto;
+    display: none;
+  }
+
+    .txtarea {
+      padding: 0;
+      width: 50%;
+      line-height: 16px;
+      resize: none;
+    }
+  
+}
+
+legend {
```

Comment:
It's better you don't override the css of HTML tags directly, because any legend will have the same stylings. How about defining a specific class to the legend you want to style?